### PR TITLE
Fix get_message when msg_id does not exist

### PR DESCRIPTION
**Pull request by paul-nameless**
Purpose of this patch is to fix bug when getting reply msg id which was deleted previously. It causes to make a request each draw call.

### CODE REVIEW

**Comment by paul-nameless** [comment]
Don't merge it, still in development.

File: tg/models.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 
 class Model:
-    def __init__(self, tg: Tdlib) -> None:
```
Comment (lightme16) [inline tg/models.py:14]: why?  we need to be consistent and either remove all constructor annotations  at once or  leave it as is :slightly_smiling_face:

File: tg/models.py [hunk 2]
```diff
@@ -306,50 +310,53 @@ def prev_msg(self, chat_id: int, step: int = 1) -> bool:
 
         return False
 
-    def get_message(self, chat_id: int, msg_id: int) -> Dict:
+    def get_message(
+        self, chat_id: int, msg_id: int
+    ) -> Optional[Dict[str, Any]]:
         msg_set = self.msg_ids[chat_id]
```
Comment (lightme16) [inline tg/models.py:316]: misleading name, it is not `msg_set` anymore, it is `msg_dict`

File: tg/models.py [hunk 3]
```diff
@@ -306,50 +310,53 @@ def prev_msg(self, chat_id: int, step: int = 1) -> bool:
...
+                self.not_found.add(msg_id)
+                return None
             return result.update
-        return next(iter(m for m in self.msgs[chat_id] if m["id"] == msg_id))
+        index = msg_set[msg_id]
+        return self.msgs[chat_id][index]
 
-    def remove_message(self, chat_id, msg_id):
+    def remove_message(self, chat_id: int, msg_id: int):
         msg_set = self.msg_ids[chat_id]
```
Comment (lightme16) [inline tg/models.py:332]: bad variable name

**Pull request closed without merging**